controllers/check_client_compatibility_test.go

```diff
@@ -0,0 +1,227 @@
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.106:36128",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.106:36802",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.107:42234",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.107:49684",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:47320",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:47388",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:58734",
+				LogGroup: "sample-cluster-client",
+			},
+		},
+		MaxProtocolClients: nil,
+		ProtocolVersion:    "fdb00b061060001",
+		SourceVersion:      "bd6b10cbcee08910667194e6388733acd3b80549",
+	},
+	{
+		ClientVersion: "6.2.15",
+		ConnectedClients: []fdbv1beta2.FoundationDBStatusConnectedClient{
+			{
+				Address:  "10.1.38.106:35640",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.106:36128",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.106:36802",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.107:42234",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.107:49684",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:47320",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:47388",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:58734",
+				LogGroup: "sample-cluster-client",
+			},
+		},
+		MaxProtocolClients: []fdbv1beta2.FoundationDBStatusConnectedClient{
+			{
+				Address:  "10.1.38.106:35640",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.106:36128",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.106:36802",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.107:42234",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.107:49684",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:47320",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:47388",
+				LogGroup: "sample-cluster-client",
+			},
+			{
+				Address:  "10.1.38.108:58734",
+				LogGroup: "sample-cluster-client",
+			},
+		},
+		ProtocolVersion: "fdb00b062010002",
+		SourceVersion:   "20566f2ff06a7e822b30e8cfd91090fbd863a393",
+	},
+}
+
+var _ = Describe("check client compatibility", func() {
+	When("validating getUnsupportedClient()", func() {
+		type testCase struct {
+			ignoredLogGroups           map[string]fdbv1beta2.None
+			expectedUnsupportedClients []string
+		}
+
+		DescribeTable("should return all the unsupported clients",
+			func(tc testCase) {
+				unsupportedClients := getUnsupportedClients(supportedVersion, "fdb00b061060001", tc.ignoredLogGroups)
```

Comment:
```suggestion
				unsupportedClients := getUnsupportedClients(supportedVersion, "fdb00b063010001", tc.ignoredLogGroups)
```

Doing a check for the 6.1 version seems wrong.